src/storage/v2/storage.hpp

```diff
@@ -402,6 +414,14 @@ class Storage final {
 
   ConstraintsInfo ListAllConstraints() const;
 
+  /// @throw std::bad_alloc
+  bool CreateSchema(LabelId primary_label, std::vector<SchemaType> &schemas_types);
+
+  /// @throw std::bad_alloc
+  bool DeleteSchema(LabelId primary_label);
```

Comment:
```suggestion
  bool CreateSchema(LabelId primary_label, std::vector<SchemaType> &schemas_types);

  bool DeleteSchema(LabelId primary_label);
```

I know that other functions mention bad alloc, but I think that is not that useful. I mean you cannot handle `bad_alloc` in any proper way in my opinion.